src/borgstore/backends/s3.py

```diff
@@ -33,13 +35,18 @@ def get_s3_backend(url):
     """
     m = re.match(s3_regex, url, re.VERBOSE)
     if m:
+        s3type = "s3" if "s3:" in url else "b2"
```

Comment:
"in url" is a bit dangerous here.

cleanest would be to give the matched string a name (like done for the profile) and then use m["s3type"].

Reply:
Agreed. Will make the change soon.